content/api/power-board.md

```diff
@@ -19,4 +19,28 @@ r.power_board.power_off()
 `power_on` is called when you setup your robot, so this doesn't need to be called manually. The ports will come on automatically as soon as your robot is ready, before the start button is pressed.
 {{% /notice %}}
 
+## Buzzing :bee:
+
+The power board has a piezo buzzer which can beep.
+
+The `buzz` function accepts multiple parameters, depending on what you want to play. The first argument is the duration of the beep, in seconds. The later arguments are either the note you want to play, or the buzzers frequency.
+
+
+{{% notice note %}}
+You should only pass `frequency` __or__ `note`. If you pass both, `note` is used, and `frequency` is silently ignored.
```

Comment:
We should do something about this!

Reply:
Well, we can change this in `robot-api`, maybe throw if both values are provided might be a nice idea

Reply:
Fixed in https://github.com/sourcebots/robot-api/pull/31